### ISSUE TITLE
EmailReadImap: fails to read utf8 filename of attachement

### ISSUE THREAD

**Issue by tennox** [n8n-io/n8n#2494]
**Describe the bug**
When it reads an email with an attachement that has an utf8 encoded filename, the attachement in n8n will have filename `=`.

**To Reproduce**
1. Send yourself an email with an attachement with a filename like `test_ä.txt`
2. Let EmailReadImap fetch the email (with `Simple` format and `Download attachements` enabled)
3. Go to 'binary tab'
4. See filename `=`

**Expected behavior**
utf-8 encoded filename should be read properly.

**Environment (please complete the following information):**
 - docker image `n8nio/n8n@904ea610d1680d745210ae21769532f0156c1a65ec513c85a6186f49db517648`
   `n8n -v` = `0.151.0`

**Additional context**

I think it's not an upstream issue, as it was already [reported and fixed many years ago](https://github.com/mscdex/node-imap/issues/86).

EML attachement with normal filename (works):
```eml
--61a4bd48_1511a1b3_2dee
Content-Type: text/plain
Content-Transfer-Encoding: base64
Content-Disposition: attachment; filename="test.pdf"
...
```
EML attachement with UTF-8 filename (broken):
```eml
--61a4ca49_4f1ac932_2dee
Content-Type: text/plain
Content-Transfer-Encoding: base64
Content-Disposition: attachment; filename="=?utf-8?Q?test=5F=C3=A4.txt?="

dGVzdC1jb250ZW50Cg==

--61a4ca49_4f1ac932_2dee--
```

**Comment by tennox** [comment]
I just figured out that this does not happen when using `Resolved` Format for EmailReadImap

**Comment by Joffcom** [comment]
Hey @tennox,

Sorry for the delay on this one, I have created an internal ticket which we will be tracking as `NODE-173` to resolve this.

**Comment by Joffcom** [comment]
Hey @tennox,

Good news, We have put in a PR which fixes this here: https://github.com/n8n-io/n8n/pull/6856

**Comment by janober** [comment]
Fix got released with [`n8n@1.2.1`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.2.1)
